### PR TITLE
Issue 239: Document an endpoint to return commitments associated with a capital project

### LINKED ISSUE #239: Open API: Document an endpoint to return commitments associated with a capital project
# Description
Create an endpoint to return commitments associated with a capital project

## Acceptance criteria
- [x] a `/capital-projects/{managingCode}/{capitalProjectid}/capital-commitments` endpoint is created under [`paths`](https://github.com/NYCPlanning/ae-zoning-api/blob/9561c9bd8b80a8ca49f52505899bfec4500904e9/openapi/openapi.yaml#L15)
  - it is placed in alphabetically within the list of paths, as practicable
- [x] summary: Find capital commitments associated with a specific capital project
- [x] operationId: findCapitalCommitmentsByManangingCodeCapitalProjectId
- [x] tags: Capital Commitments
- [x] parameters (#235) 
   - managingCode
   - capitalProjectId
- [x] `get` method
   - `200` response returns a body with parameters:
      -  capitalCommitments: list of CapitalCommitment #232
   - `400` response returns a BadRequest
   - `404` response returns a NotFound
   - `500` response returns an InternalServerError

## Helpful Code Snippet
[/zoning-districts/{id}/classes](https://github.com/NYCPlanning/ae-zoning-api/blob/9561c9bd8b80a8ca49f52505899bfec4500904e9/openapi/openapi.yaml#L217)

## Related tickets
blocked by #232, #235

### PR DESCRIPTION
**Pull request by horatiorosa**
#### Description:
239 document capital project commitments endpoint

#### Tickets
 - Closes #239

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -88,7 +88,35 @@ paths:
         '404':
           $ref: "#/components/responses/NotFound"
         '500':
-          $ref: "#/components/responses/InternalServerError" 
+          $ref: "#/components/responses/InternalServerError"
+  /capital-projects/{managingCode}/{capitalProjectid}/capital-commitments:
+    get:
+      summary: Find capital commitments associated with a specific capital project
+      operationId: findCapitalCommitmentsByManangingCodeCapitalProjectId
```
Comment (TangoYankee) [inline openapi/openapi.yaml:95]: Spelling `Mananging`

**Review by TangoYankee — CHANGES_REQUESTED** [review]

File: openapi/openapi.yaml (hunk 1 above)
Comment (horatiorosa) [reply openapi/openapi.yaml:95]: that what I get for copy/pasting

😩

Comment (TangoYankee) [reply openapi/openapi.yaml]: Lol, I just checked the original ticket. I'm blaming my keyboard for the typo in the ticket

**Review by TangoYankee — COMMENTED** [review]
The code changes meet spec. Waiting for its turn in the queue of endpoints (#278 ) before approval.

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -88,7 +88,35 @@ paths:
         '404':
           $ref: "#/components/responses/NotFound"
         '500':
-          $ref: "#/components/responses/InternalServerError" 
+          $ref: "#/components/responses/InternalServerError"
+  /capital-projects/{managingCode}/{capitalProjectid}/capital-commitments:
+    get:
+      summary: Find capital commitments associated with a specific capital project
```
Comment (TangoYankee) [inline openapi/openapi.yaml:94]:
```suggestion
      summary: 🚧 Find capital commitments associated with a specific capital project
```

**Review by TangoYankee — CHANGES_REQUESTED** [review]

File: openapi/openapi.yaml [hunk 3]
```diff
@@ -135,6 +137,15 @@ paths:
           $ref: '#/components/responses/BadRequest'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /city-council-districts:
```
Comment (pratishta) [inline openapi/openapi.yaml:140]: Why is this endpoint also being documented in this PR? This should be a part of main so if you rebase off it, it should not show up as a new addition for this PR

Comment (horatiorosa) [reply openapi/openapi.yaml:140]: I moved it from 114 to 149 after capital-projects to alphabetize it

Comment (pratishta) [reply openapi/openapi.yaml:140]: Gotcha

**Review by pratishta — APPROVED** [review]
LGTM

**Pull request merged**